Corrects wrong links for powershell SF commands

murdockcrc opened the pull request:
`Disable-ServiceFabricNode` and `Get-ServiceFabricNode` were pointing to the wrong links in the docs.

azurecla commented:
Hi __@stefsch__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (stefsch). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
Corrects wrong links for powershell SF commands

`Disable-ServiceFabricNode` and `Get-ServiceFabricNode` were pointing to the wrong links in the docs.

Commit:
Update service-fabric-cluster-scale-up-down.md

removes reference to en-us localization.
